src/main.ts

```diff
@@ -20,6 +21,7 @@ import '@/assets/styles/typography.scss'
 import '@/assets/styles/inputs.scss'
 import '@/assets/styles/components.scss'
 import '@/assets/styles/transitions.scss'
+import { PATH_CONFIG_DEFAULT } from './constants.js'
```

Comment:
How come this references a `.js` file?

Reply:
VS Code adds the extension when auto-completing imports and I forgot to remove it. Preferably, I would use the .ts extension, but TypeScript forbids this still.